src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+document.getElementById('email-form').addEventListener('submit',
+  function(event) {
+    event.preventDefault();
+    event.stopPropagation();
+
+    const input = document.getElementsByClassName('main__nav-email')[0];
```

Comment:
You are using 'getElementsByClassName' which returns a live HTMLCollection of elements with the given class name. You then immediately access the first element. It would be better to use 'querySelector' instead, which returns the first Element within the document that matches the specified selector, or group of selectors.